### PR TITLE
Fixes #20849 - Add more information about Arf Report

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by ares** [comment]
@xprazak2 could you please rebase?

**Comment by xprazak2** [comment]
Rebased.

File: lib/hammer_cli_foreman_openscap/arf_report.rb [hunk 1]
```diff
@@ -6,8 +6,16 @@ class ArfReport < HammerCLIForeman::Command
     class ListCommand < HammerCLIForeman::ListCommand
       output do
         field :id, _("Id")
-        field :host, _("Host")
         field :reported_at, _("Reported at")
+        from :host do
+          field :name, _("Host name")
+        end
+        from :openscap_proxy do
+          field :name, _("Openscap proxy name")
```
Comment (ares) [inline lib/hammer_cli_foreman_openscap/arf_report.rb]: s/Openscap/OpenSCAP/, will fix on merge

**Comment by ares** [comment]
Thanks @xprazak2, merged as 87a77d01640869c18862994be5b8c1eb61a5546d

**Pull request closed without merging**